pages/docs/database-connections/role-based-credentials.mdx

```diff
@@ -0,0 +1,70 @@
+---
+description: Role-based data credentials allow you to control what users can access from the same data connection.
+---
+
+import { Callout } from "nextra-theme-docs";
+
+# Role-Based Database Credentials
+
+All Hashboard data connections can be optionally configured to use additional credentials which can be assigned to users on a per-role basis. 
+
+After setting up database-level permissions within the data warehouse you use with Hashboard, these credentials can be added within Hashboard and then assigned to your Hashboard project's roles.
+
+This enables projects to control what data users can access when viewing models, saved explorations, and dashboards built from the same data connection.
```

Comment:
This reads a bit confusing to me - are you saying that this will enable the specific roles in projects? Right now it reads that there will be multiple projects.

Reply:
+1, should instead read "This enables you to control..."